rfc/rfc021-vp_token-grant-type.md

```diff
@@ -132,20 +136,67 @@ The second paragraph describes the requirements that apply to JWT encoded VP's.
 5. The `verificationMethod` field of the Proof MUST be a DID URL.
 6. The `holder` field if present MUST match the `credentialSubject.id` field from all the Verifiable Credentials that are used to request the access token.
```

Comment:
maybe we should omit this, if the verification method ID already specifies the holder? Otherwise, it only complicates validation.